tests/acceptance/README.md

```diff
@@ -0,0 +1,24 @@
+Acceptance tests
+================
+
+The tests will run on a file generated by the reporter.
+The file is generated using Mugshot, so you first need to run a browser:
+
+```sh
+npm run browser
+```
+
+Then you can generate the report. Two tests are made to fail.
```

Comment:
But then the state of the test would be 'passed' and will not generate the diff view 😞